src/firefly/js/visualize/ui/VisCtxToolbarView.jsx

```diff
@@ -94,65 +86,104 @@ function changeAutoConvert(pv, auto) {
     if (auto) doHiPSImageConversionIfNecessary(nextPv);
 }
 
-
-const projOptions= [
-    {label: 'HiPS', value: 'sin', tooltip: 'All-sky multi-resolution picture with spherical projection, up to 180 degrees'},
-    {label: 'HiPS/Aitoff', value: 'aitoff', tooltip: 'All-sky multi-resolution picture with AITOFF projection, up to 360 degrees'},
-];
-const defHFOptions= [
-    {label: 'FITS', value: 'fits', tooltip: 'Scientific pixel data over limited regions'},
-    ...projOptions
-];
-
-function HipsFitsConvertButton({pv}) {
+function HipsOptionsDropdown({pv}) {
+    const AITOFF_TEXT= 'Aitoff';
+    const SPHER_TEXT= 'Spherical';
     const plot= primePlot(pv);
     if (!plot) return undefined;
+    const {plotId}= plot;
+    const csysText= plot.imageCoordSys===CoordinateSys.GALACTIC ? 'Gal' : 'Eq J';
```

Comment:
I don't think "Eq J" is going to be very meaningful.  The "J" makes it more difficult to understand, not less.

If you are certain that "Equatorial / Spherical" is not going to work, then "Equ" is IMO a better abbreviation than "Eq J".  But it's still pretty cryptic if you don't already know what the purpose of the menu is.

Reply:
I can change it, we probably need @lrebull... but you don't think an astronomer would know what Eq J means?